eduVPN/ProviderService.swift

```diff
@@ -1052,7 +1052,8 @@ class ProviderService {
                     if let certificate = certificate, let privateKey = privateKey {
                         // Import into Keychain
                         let passphrase = String.random()
-                        configurationService.createPKCS12(certificate: certificate, privateKey: privateKey, passphrase: passphrase) { result in
+                        let friendlyName = configFileURL.lastPathComponent
```

Comment:
What is this? lastPathComponent? The point was to indicate the domain name of the VPN service the configuration cam from, but maybe that is what this does?

Reply:
This is for a local config.

Reply:
Ah! :)